.gitignore

```diff
@@ -1 +1,3 @@
 __pycache__/
+pythonenv*/
+build/
```

Comment:
Unsure where `build/` flew out from, but the `pythonenv*/` entry should exist in your [user-local](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file) or [user-global](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) gitignore file.